src/core/java/com/enderio/core/client/item/ItemBarRenderer.java

```diff
@@ -0,0 +1,36 @@
+package com.enderio.core.client.item;
+
+import com.mojang.blaze3d.systems.RenderSystem;
+import com.mojang.blaze3d.vertex.*;
+import net.minecraft.client.renderer.GameRenderer;
+import net.minecraft.util.FastColor;
+
+public class ItemBarRenderer {
+
+    public static boolean renderBar(float fillRatio, int xOffset, int yOffset, float blitOffset, int color) {
```

Comment:
return nothing, if renderstate is reset to enableBlend and enable texture you can return false in the Decorator

Reply:
Done :) Now all of them return false and the renderBar method returns nothing.